omero/sysadmins/unix/server-linux-walkthrough.txt

```diff
@@ -72,8 +78,39 @@ Nginx configuration directory, and disable the default configuration:
 .. literalinclude:: walkthrough/step05_ubuntu1404_nginx.sh
 
 | :download:`step05_ubuntu1404_nginx.sh <walkthrough/step05_ubuntu1404_nginx.sh>`
-| :download:`step05_centos6_nginx.sh <walkthrough/step05_centos6_nginx.sh>`
+| :download:`step05_centos7_nginx.sh <walkthrough/step05_centos7_nginx.sh>`
+
+Installing Apache
+-----------------
+
+As **root** install Apache 2.4::
+
+	apt-get -y install apache2 libapache2-mod-wsgi
+
+As the **omero** system user, configure OMERO.web:
+	
+.. literalinclude:: walkthrough/setup_omero_apache24.sh
+
+When installing on Ubuntu, edit the Apache OMERO configuration ``apache.conf.tmp``
+to set ``WSGISocketPrefix``::
```

Comment:
We need it in all cases. Just say explicitly where exactly placing sockets and lock files used by Apache modules. Also add note to not put the sockets in the system temporary working directory.